src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,128 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connecting a libSQL database for production
+
+Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server
+
+Details of the libSQL connection (e.g. encryption key, replication, sync interval) can be configured as query parameters in the remote connection URL.
+
+For example, to have a local file work as an embedded replica to an encrypted libSQL server, you can set the following environment variables:
+
+```dotenv title=".env"
+ASTRO_DB_REMOTE_URL=file://local-copy.db?encryptionKey=your-encryption-key&syncInterval=60&syncUrl=libsql%3A%2F%2Fyour.server.io
+ASTRO_DB_APP_TOKEN=token-to-your-remote-url
+```
+
+<ReadMore>See all available [remote URL configuration options](#remote-url-configuration-options)</ReadMore>
+
+### Deploy your project with a live connection
+
+You can deploy your Astro DB project with a live connection to your libSQL-compatible database. This is possible with any deployment platform using any static or server rendering mode.
+
+Configure your build command to connect to your libSQL-compatible database using the `--remote` flag. This example applies the flag to a `"build"` script in your project's `package.json`. If your deployment platform accepts a build command, ensure this is set to `npm run build`.
+
+```json title="package.json" "--remote"
+{
+  "scripts": {
+    "build": "astro build --remote"
+  }
+}
+```
```

Comment:
It is worth pointing out somewhere around this section that `build --remote` uses the connection to the remote DB both locally during the build and on the server, so the environment variables have to be available in both.

This was simpler when using Studio because it used the local logged-in user from the CLI, so the token env var was only needed on the server.